### DISCUSSION TITLE
Autotranslate plugin

### DISCUSSION THREAD

**Discussion by xperiandri** [OrchardCMS/OrchardCore#11193] [Q&A]
Is there a plugin that can call Bing/Google/Promt/Abbeyy API to automatically translate all strings on a content item localization creation?

**Comment by Skrypt** [comment]
Nothing that I know of about this. I'm not sure if we have the appropriate event in the IContentLocalizationPartHandler.

/cc @jptissot

**Comment by hishamco** [comment]
This will be cool if OC provide a UI for manage the localized strings, but for now creating a tool or extend PO Extractor will be the best option

FYI I did a resource generator tool 5 years ago that supports translation https://github.com/hishamco/LocalizationResourceGenerator may be we could add such service as external module
